src/ast/mod.rs

```diff
@@ -477,6 +477,17 @@ pub enum CastKind {
     DoubleColon,
 }
 
+/// The syntax used in a CEIL or FLOOR expression.
```

Comment:
Whatcan you add the context about coming from kinesis and a link to the docs as well in these doc comments?